### DISCUSSION TITLE
Transferring Mint through Phantom

### DISCUSSION THREAD

**Discussion by react-universe** [phantom/docs#276] [Help & Support]
### Summary

I want to transfer mint(Bonk) through Phantom Wallet to my address. I am using React js and solana/web3.js, solana/spl-token packages that help me to for transactions. Solana transfer through Phantom working perfectly fine. But when I started transferring Bonk token I faced the issues. I did a search and implemented everything that made the code work, But now when I do transaction, Phantom opens and doesn't show the token that I am transferring.

```
export const createTransferTransaction = async (
  publicKey,
  connection,
  amount,
  toPublicKey,
  mintAddress,
  numberDecimals = 5
) => {
  const transaction = new Transaction();
  const fromPublicKey = new PublicKey(publicKey);
  const toPublicKeyInstance = new PublicKey(toPublicKey);

  // Existing BONK token mint address
  const BONK_MINT_ADDRESS = "DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263";

  const fromTokenAccount = await getAssociatedTokenAddress(
    new PublicKey(BONK_MINT_ADDRESS),
    fromPublicKey
  );

  const toTokenAccount = await getAssociatedTokenAddress(
    new PublicKey(BONK_MINT_ADDRESS),
    toPublicKeyInstance
  );

  transaction.add(
    createTransferInstruction(
      fromTokenAccount,
      toTokenAccount,
      fromPublicKey,
      10 * Math.pow(10, 5) // transferring 10 Bonk
    )
  );

  transaction.feePayer = fromPublicKey;

  const { blockhash } = await connection.getLatestBlockhash();
  transaction.recentBlockhash = blockhash;

  return transaction;
};

```
As shown in pic, it doesn't show transaction is for Bom and also when I open advanced, it shows wrong amount that I calculated.

![image](https://github.com/user-attachments/assets/cec5baca-0aeb-499d-9900-56bb17e7c990)

### Steps to Reproduce

Every time i am getting same issue. 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @react-universe,

We don't provide code trouble shooting. From a quick glance, I think you're attempting to send tokens from the bonk mint which you don't have signing authority for.
